src/Bicep.Core/Registry/Oci/OciAnnotations.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+
+namespace Bicep.Core.Registry.Oci
+{
+    /// <summary>
+    /// Additional information provided through arbitrary metadata.
+    /// </summary>
+    public class OciAnnotations
+    {
+        public OciAnnotations(string documentationUrl)
+        {
+            this.DocumentationUrl = documentationUrl;
+        }
+
+        /// <summary>
+        ///  URL to get documentation on the image.
+        /// </summary>
+        public string DocumentationUrl { get; }
```

Comment:
>DocumentationUrl [](http://example.com/codeflow?start=22&length=16)

Isn't this going to serialize as "documentationUrl" in the manifest? Shouldn't we serialize this as `org.opencontainers.image.documentation` which is the standard OCI annotation for documentation URL.

We can declare our own, but we should generally use existing well-known ones from this list first: [https://github.com/opencontainers/image-spec/blob/main/annotations.md](https://github.com/opencontainers/image-spec/blob/main/annotations.md)

Reply:
It should be fairly easy to customize the property name in the serializer we use, though.

Reply:
@majastrz, do you mean manifest should look like below?
![image](https://user-images.githubusercontent.com/30270536/214720076-e930291a-cf39-417f-bce1-13fbf3a02ba2.png)

Reply:
No, annotations are just like a `Dictionary<string,string>`, so the key would be `org.opencontainers.image.documentation`. They have namespaces so different orgs or tools can easily introduce their own if they need to and avoid clashes.

Reply:
Ah got it. Thanks for the explanation. I'll update it. Thanks!

Reply:
Updated here:
https://github.com/Azure/bicep/commit/63d578fb3a0e0a3ec200984176f38dad740d79e5

Thanks!